src/experiments/experiments.jl

```diff
@@ -175,6 +184,9 @@ end
 
 
 function mkworkspace(exp::Experiment)
+    if !exp.save
+        return
+    end
```

Comment:
I believe it is unnecessary due to [this if statement](https://github.com/Pidoh/Pidoh.jl/pull/16/files#diff-c604085a8194a47bc3c034dbdedac758c2bde3894a08454bf5e2fd49303e843eR74-R76).

Reply:
Thanks! removed.